src/Twilio.Api/TransportException.cs

```diff
@@ -5,11 +5,11 @@
 
 namespace Twilio
 {
-    public class TransportException<T> : Exception

```

Comment:
@chris-smith-zocdoc is the type information here useful? Does it get print out as part of the Exception? 
`DeleteMessage()` is using a none generic version of `Execute`.